enhancements/metering/slim-install.md

```diff
@@ -0,0 +1,210 @@
+---
+title: slim-install
+authors:
+  - "@timflannagan1"
+reviewers:
+  - TBD
+approvers:
+  - @operator-framework/team-metering
+  - @bparees
+creation-date: 2020-10-06
+last-updated: 2020-10-06
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Slim Installation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Due to the heavy footprint of the current Hadoop-based metering stack, it's difficult to install in smaller clusters, like a demo or developer environment, which is affecting potential adoption and opens the door to replicating existing metering functionality, with an in-house solution. This proposal outlines a path towards a slimmer metering stack, both in terms of reducing complexity and the overall resource footprint, and is centered around replacing the properties that the Hadoop, Hive, and Presto components offer, with Postgresql acting as both the underlying storage and compute engine. This will allow metering to set more reasonable defaults and reduces the barrier of entry when attempting to install metering on smaller or medium-sized environments as it requires little-to-none user intervention.
```

Comment:
Think it would improve clarity to say `Metering Operator (Metering)` once at start and capitalize the `M` in `Metering` throughout the doc.

Reply:
That's fine with me. I'll update it.